src/compute-engine/global-types.ts

```diff
@@ -1122,6 +1122,9 @@ export interface BoxedExpression {
    *
    * :::info[Note]
    * Applicable to canonical and non-canonical expressions.
+   * 
+   * To specify a match for single symbol (non wildcard), it must be boxed (e.g. `{ match:
+   * ce.box('x'), ... }`), but it is suggested to use method *'subs()'* for this.
```

Comment:
This in reference to opening-PR message: this is the current 'workaround'
(not that it matters since `subs()` can be used)